Fw/Python/src/fprime/common/models/serialize/time_type.py

```diff
@@ -19,14 +19,15 @@
 from __future__ import print_function
 from __future__ import absolute_import
 
-import time
-from datetime import *
-from pytz import *
+import datetime
+
+# from pytz import TimeBase
```

Comment:
Why is this commented out?

Reply:
I was testing to see if it was needed, turns out it wasnt and I forgot to delete